### PR TITLE
extract heated floor area data

### PR DESCRIPTION
**Pull request by buckley-w-david**
This PR adds functionality to extract the above and below grade heated floor area data.

https://trello.com/c/q0fW8AUX/112-extract-load-heated-floor-data

### CODE REVIEW

**Comment by buckley-w-david** [comment]
[Extract/Load Heated Floor data](https://trello.com/c/q0fW8AUX/112-extract-load-heated-floor-data)

File: python/src/energuide/snippets.py [hunk 1]
```diff
@@ -108,20 +108,29 @@ def _window_snippet(window: etree.ElementTree) -> typing.Dict[str, typing.Any]:
...
+    }
+
 
 def snip_house(house: etree.ElementTree) -> typing.Dict[str, typing.Any]:
     ceilings = house.findall('Components/Ceiling')
     floors = house.findall('Components/Floor')
     walls = house.findall('Components/Wall')
     doors = house.findall('Components//Wall/Components/Door')
     windows = house.findall('Components//Components/Window')
+    specifications = house.find('Specifications')
```
Comment (JasonMWhite) [inline python/src/energuide/snippets.py]: I think it should be `.find('Specifications/HeatedFloorArea')` directly here.

File: python/src/energuide/snippets.py [hunk 2]
```diff
@@ -64,19 +64,28 @@ def _window_snippet(window: etree._Element) -> typing.Dict[str, typing.Optional[
     })
 
 
+def _heated_floor_area_snippet(heated_floor_area: etree._Element) -> typing.Dict[str, typing.Any]:
```
Comment (JasonMWhite) [inline python/src/energuide/snippets.py]: I think we can change the return type to be a bit more specific.

File: python/tests/test_snippets.py [hunk 3]
```diff
@@ -82,6 +82,13 @@ def test_window_snippet(house: etree.ElementTree) -> None:
     }
 
 
+def test_heated_floor_area_snippet(house: etree.ElementTree) -> None:
```
Comment (JasonMWhite) [inline python/tests/test_snippets.py]: `house` is actually `etree._Element`. My PR (#109)  updates the rest of these to that input type.

**Pull request merged**